lib/publiccloud/gce.pm

```diff
@@ -81,8 +83,66 @@ sub upload_img {
     return $img_name;
 }
 
+sub ipa {
+    my ($self, %args) = @_;
+
+    $args{instance_type}        //= 'n1-standard-2';
+    $args{cleanup}              //= 1;
+    $args{ssh_private_key_file} //= '.ssh/id_rsa';
+    $args{tests}                //= '';
+    $args{timeout}              //= 60 * 20;
+    $args{results_dir}          //= 'ipa_results';
+
+    $args{tests} =~ s/,/ /g;
+
+    if (script_run('test -f ' . $args{ssh_private_key_file}) != 0) {
+        assert_script_run('SSH_DIR=`dirname ' . $args{ssh_private_key_file} . '`; test -d $SSH_DIR || mkdir -p $SSH_DIR');
+        assert_script_run('ssh-keygen -b 2048 -t rsa -q -N "" -f ' . $args{ssh_private_key_file});
+    }
+
+    my $cmd = 'ipa --no-color test gce ';
+    $cmd .= '--debug ';
+    $cmd .= '--service-account-file "' . $credentials_file . '" ';
+    $cmd .= '--distro sles ';
```

Comment:
Lets make it as a parameter like explained here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/provider.pm#L65
I see that I also did it wrong in `publiccloud::azure.pm`. We should fix that up there too.

Reply:
ok